scripts/create-package/plop-templates-node/just.config.ts

```diff
@@ -0,0 +1,5 @@
+const { preset } = require('@uifabric/build');
+
+preset();
+
+task('build', 'build:node-lib').cached();
```

Comment:
remove?

Reply:
No this is correct--for node libraries we want to only build commonjs and have it output to lib, which is what this does.

Reply:
yeah i read it wrong :)